core/src/test/java/org/infinispan/statetransfer/RemoteGetDuringStateTransferTest.java

```diff
@@ -54,40 +63,88 @@
+   | [4]    |     | T2            | 1: *, 2:  *       | T0/T1              |       |                   |                    |
+   | 2112   | 8.1 | T2            | 1:T1, 2: T1       | T2/T3/T4           | N1    |                   |                    |
+   | 2122   | 8.2 | T2            | 1:T1, 2: T2       | T2/T3/T4           | N1+2  |                   |                    |
+   | 2132   | 8.3 | T2            | 1:T1, 2: T3/T4    | T2/T3/T4           | N1+2  |                   |                    |
+   | 2212   | 8.4 | T2            | 1:T2, 2: T1       | T2/T3/T4           | N1    |                   |                    |
+   | 2222   | 8.5 | T2            | 1:T2, 2: T2       | T2/T3/T4           | N1+2  |                   |                    |
+   | 2232   | 8.6 | T2            | 1:T2, 2: T3/T4    | T2/T3/T4           | N1+2  |                   |                    |
+   | 2312_22| 9.1 | T2            | 1:T3/T4, 2: T1    | T2/T3/T4           | Y     | 2: T2             | T2/T3/T4           |
+   | 2312_32| 9.2 | T2            | 1:T3/T4, 2: T1    | T2/T3/T4           | Y     | 2: T3/T4          | T2/T3/T4           |
+   | 2322   | 8.7 | T2            | 1:T3/T4, 2: T2    | T2/T3/T4           | N2    |                   |                    |
+   | 2332   | 8.8 | T2            | 1:T3/T4, 2: T3/T4 | T2/T3/T4           | N2    |                   |                    |
+   | [2]    |     | T3            | 2: T0/T1          |                    |       |                   |                    |
+   | [4]    |     | T3            | 2: T2             | T0/T1/T2           |       |                   |                    |
+   | 323    | 10  | T3            | 2: T2             | T3/T4              | N2    |                   |                    |
+   | [4]    |     | T3            | 2: T3/T4          | T0/T1/T2           |       |                   |                    |
+   | 333    | 11  | T3            | 2: T3/T4          | T3/T4              | N2    |                   |                    |
+   | [2]    |     | T4            | 2: T0/T1/T2       |                    |       |                   |                    |
+   | [4]    | 12  | T4            | 2:T3              | T0/T1/T2/T3        | N2    |                   |                    |
+   | 434    | 12  | T4            | 2:T3              | T4                 | N2    |                   |                    |
+
+   *) The retry will go to both node 1 and 2 but 1 in T3 will respond with UnsureResponse
    [1] too similar to the previous scenario
-   [2] impossible because we can't have T0 on one node and T2 on another node at the same time
-   [3] impossible, retry is executed only after we allow the topology update
+   [2] impossible because we topologies can't differ by more than 1 at the same time
    [4] impossible, first response was received in later topology than second response
```

Comment:
Since the table is much bigger than before, maybe we should remove the `[4]` rows altogether?